HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectType.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class MapObjectType
+    {
```

Comment:
Putting colors in the type seems somewhat odd, since it looks like the MapObjectBasics class was generally intended to contain properties of the room that aren't strictly related to drawing? I would consider defining a ColorProvider interface that takes the type as an argument and returns a color, although using that without a DI container might be inconvenient. Definitely consider moving the color information elsewhere eventually though.

Reply:
In the meantime, MapObjectBasics has been converted into another class and there has been a generalization introduced, so there are some differences from the current state. I'll leave the colors here for now. I believe that this works quite as intended for now and probably for the future. If necessary, refactoring will be done in the future.